src/lightcone/ethereum/metaMask.js

```diff
@@ -152,6 +163,39 @@ export async function contractWalletValidate2(web3, account, msg, sig) {
   });
 }
 
+// Authereum account contract hashes the data in the validation function,
+// so we must send the data plain text.
+export async function contractWalletValidate3(web3, account, msg, sig) {
```

Comment:
Is this only for Atheneum?

Reply:
as far as I know it's only for Authereum ([implementation](https://github.com/authereum/contracts/blob/master/contracts/account/ERC1271Account.sol#L29)). There's possibility that other contract wallets implement isValidSignature in this similar way (before eip1271 became [more standardized](https://github.com/ethereum/EIPs/issues/1271)) but I haven't looked at them

Reply:
@miguelmota I will merge the PR and deploy it to Loopring.io in the next release. ETA: 2-3 days.

Reply:
Thanks!

Reply:
Deployed to loopring.io